test/parallel/test-http-pipeline-regr-3332.js

```diff
@@ -26,15 +26,8 @@ const server = http.createServer(function(req, res) {
 
   // Just let the test terminate instead of hanging
   client.on('close', function() {
-    if (received !== COUNT)
+    if (countdown.remaining !== 0)
       server.close();
```

Comment:
@sreepurnajasti do you know why is this needed? Isn't the `server.close()` above sufficient?

Reply:
@lpinca Thanks for the review. This block is used to close the server instead of hanging. However, conditional check may not be required. If it is also, its fine. So, let me know whether to make any changes!!

Reply:
The whole `client.on('close')` block should be removed. Since there's now no `process.on('exit')` check, we should let the test timeout if this isn't working correctly which would then signal that the test is broken.

Reply:
@lpinca @apapirovski Updated. Please review.